src/Model/Contact.php

```diff
@@ -109,6 +109,16 @@ class Contact extends BaseObject
 	 * @}
 	 */
 
+	/**
+	 * @param  integer       $id
+	 * @return array|boolean Contact record if it exists, false otherwise
+	 * @throws \Exception
+	 */
+	public static function getById($id)
```

Comment:
I would like to add such functions only when they contain some additional functionality (like the similar named function in the ```User``` class). And if we have the need for encapsulating some stuff, I would prefer the way like we did in the ```Item``` class. (Not the uglyness of the code there, but using the same function names like in the database class)